### ISSUE TITLE
Training data 

### ISSUE THREAD

**Issue by xuliwalker** [2xic/openpose-pigs#2]
Hi! I am a student from Singapore University of Technology and Design, and I am recently working on a project of animal pose estimation and counting. I am interested in your work of pig pose estimation and I have some questions about the label format of your training data and training steps. We have collected similar data, but I am not sure how to train our model on the collected data. Since I cannot connect you by the provided email, I open an issue here and I wonder could you please share the training data and training steps for my reference. Thanks for your help! You can reach me at li_xu@mymail.sutd.edu.sg. :)

**Comment by 2xic** [comment]
Hey,

The training data is not public and I will not be able to share that with you because of legal reasons. You might be able to get a sample if you send a reasonable request to `oyvind.nordbo@norsvin.no`, he was the one we had dialog with regarding the data and represents the company that created the data.

I sent you an email, and if you give me some more information about your project I might still be able to give some tips regarding annotation of the data and training of the model.
